app/scripts/views/settings/clients.js

```diff
@@ -178,7 +176,7 @@ define(function (require, exports, module) {
     openPanel () {
       this.logViewEvent('open');
       // manually submit using an element to trigger the progress indicator mixin
-      this.$el.find('.clients-refresh').trigger('submit');
+      return this.validateAndSubmit();
```

Comment:
Instead of involving the DOM, we can just call `validateAndSubmit` which does the exact same thing and makes testing a heck of a lot easier.